internal/wire/parse.go

```diff
@@ -880,16 +880,27 @@ func processFieldsOf(fset *token.FileSet, info *types.Info, call *ast.CallExpr)
 		return nil, notePosition(fset.Position(call.Pos()),
 			errors.New("call to FieldsOf must specify fields to be extracted"))
 	}
+	firstArgReq := "first argument to FieldsOf must be a pointer to a struct or a pointer to a pointer to a struct; found %s"
```

Comment:
nit: Make this a constant:

```suggestion
	const firstArgReqFormat = "first argument to FieldsOf must be a pointer to a struct or a pointer to a pointer to a struct; found %s"
```